src/control.rs

```diff
@@ -81,10 +82,8 @@ pub fn unset_override() {
     SHOULD_COLORIZE.unset_override()
 }
 
-lazy_static! {
 /// The persistent [`ShouldColorize`].
-    pub static ref SHOULD_COLORIZE: ShouldColorize = ShouldColorize::from_env();
-}
+pub static SHOULD_COLORIZE: Lazy<ShouldColorize> = Lazy::new(|| ShouldColorize::from_env());
```

Comment:
That can be changed into this (the CI will complain about it once it runs as well):

```rust
pub static SHOULD_COLORIZE: Lazy<ShouldColorize> = Lazy::new(ShouldColorize::from_env);
```